app/views/admin/statistics_announcements/show.html.erb

```diff
@@ -11,7 +11,7 @@
         <%= @statistics_announcement.title %>
       </span>
     </h1>
-    <%= link_to "View announcement on GOV.UK", statistics_announcement_url(@statistics_announcement, public_and_cachebusted_url_options) %>
+    <%= link_to "View announcement on GOV.UK", @statistics_announcement.public_url(public_and_cachebusted_url_options) %>
```

Comment:
I wonder if here we should be using the `view_on_website_link_for` helper rather than specifying this link like this. It will then use the public_url method under the hood, and will make things it more consistent - this was the only case I could find outside of that helper file of `public_and_cachebusted_url_options` being called directly outside of that helper, so might be a useful thing for when we eventually rationalise that helper file.